docs/commands/SetParameters.md

```diff
@@ -0,0 +1,99 @@
+# SetParameters
+
+Parameters setup command. The sensor confirms the execution of the command with [response](#response)
+Attention! Command is not fully documented!
+
+
+## Request
+
+### Request format
```

Comment:
```suggestion
### format
```

Reply:
fixed to `### Format` - same at other commands